dpp/src/block_time_window/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod validate_time_in_block_time_window;
```

Comment:
Are those changes from https://github.com/dashevo/rs-platform/pull/138? I think we need to merge #138 first then before reviewing this one